pvnet_app/app.py

```diff
@@ -514,6 +532,24 @@ def app(
             update_gsp=True,
             apply_adjuster=apply_adjuster,
         )
+        
+        if save_gsp_sum:
+            # Save the sum of GSPs independently - mainly for summation model monitoring
+            sql_forecasts = convert_dataarray_to_forecasts(
+                da_abs_sum_gsps, 
+                session, 
+                model_name=gsp_sum_model_name_ocf_db, 
+                version=pvnet_app.__version__
+            )
+
+            save_sql_forecasts(
+                forecasts=sql_forecasts,
+                session=session,
+                update_national=True,
+                update_gsp=False,
+                apply_adjuster=apply_adjuster,
```

Comment:
could apply_adjuster be false for this? Unless you particular want to see it with the adjuster

Reply:
Sure